README.md

```diff
@@ -75,4 +79,41 @@ It consists of 1 robot:
 * `start_special_dissemination`: sends objects with content type `webarchive-binary` to wasCrawlDisseminationWF.
 
 ## Index rollup
-There is a scheduled task to roll up the `level0.cdx` and `level0.cdxj` files into `level1` each night, plus additional rollups to `level2` and `level3`, monthly and yearly respectively. 
+There is a scheduled task to roll up the `level0.cdx` and `level0.cdxj` files into `level1` each night, plus additional rollups to `level2` and `level3`, monthly and yearly respectively.
+
+
+# Prerequisites
+
+## For thumbnail image creation
+
+1. Kakadu Proprietary Software Binaries - for JP2 generation
+2. libvips
+3. Exiftool
+
```

Comment:
We also need imagemagick in this list until ThumbnailGeneratorService is updated to use libvips

Reply:
I'm working on it.  I discovered we had specs tagged with image_prerequisite that tested the code I was interested in.  And one depends on a VCR cassette.  So I'm changing them to be run all the time.  I should have had this as a draft first.

Reply:
I just created issue #497 which already has WIP (Tony C and I made some headway today) on getting rid of minimagic.